modules/runtime/nuxeo-runtime/src/main/java/org/nuxeo/runtime/model/impl/RegistrationInfoImpl.java

```diff
@@ -72,8 +72,12 @@
     @XNode("@name")
     ComponentName name;
 
-    @XNode("@disabled")
-    boolean disabled;
+    @XNode("@disable")
+    protected boolean disable = false;
+
+    // @since 11.5
+    @XNode("@enable")
+    protected boolean enable = true;
```

Comment:
You could have only one field, for instance `enable` like elsewhere, and a setter for `@disable` that sets `enable = !disable`. The API could just have `isEnabled` and no redundant logic needed like `ri.isDisabled() || !ri.isEnabled()`

Reply:
Indeed but that would imply sorting properties (xmap supports it) to make sure disable is set after enable, and i was trying not to put logics inside the descriptor, as we're trying to stick to that "Data only object" rule.
We can still discuss it if it's better.